app/code/Magento/Backend/Block/Widget/Grid/Extended.php

```diff
@@ -1252,6 +1260,16 @@ public function getCollection()
         return $this->_collection;
     }
 
+    /**
+     * get collection object
+     *
+     * @return \Magento\Framework\Data\Collection
+     */
+    public function getUnloadedCollection()
```

Comment:
It's a breaking change. Please look into the guide https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/

Reply:
Hi @slavvka 

See my comment in https://github.com/magento/magento2/issues/12594#issuecomment-421357028 
How do you see a solution for this change? Because both won't be good enough, I think.